node/risk-app/server/models/rivers-and-sea.js

```diff
@@ -0,0 +1,21 @@
+function RiversAndSeaModel (riskProbability, address, backLink) {
+  let riskColour
+
+  if (riskProbability === 'High') {
+    riskColour = 'red'
+  } else if (riskProbability === 'Medium') {
+    riskColour = 'orange'
+  } else if (riskProbability === 'Low') {
+    riskColour = 'yellow'
+  } else {
+    riskColour = ''
+  }
```

Comment:
Just realised I called the SCSS file that I created for this rivers and sea but it can be used for all of the new pages so I will update this in the groundwater ticket I am doing now.